static/js/publisher/form/banner.test.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+
+import Banner from "./banner";
+
+const backgroundSelector = ".p-market-banner__image img";
+
+function renderBanner(bannerImage, updateState, restrictions) {
+  if (bannerImage && updateState) {
+    return render(
+      <Banner bannerImage={bannerImage} updateImageState={updateState} />
+    );
+  }
+  if (bannerImage && updateState) {
+    return render(
+      <Banner bannerImage={bannerImage} updateImageState={updateState} />
+    );
+  }
+  if (updateState) {
+    return render(<Banner updateImageState={updateState} />);
```

Comment:
Isn't it equivalent of `render(<Banner bannerImage={undefined} updateImageState={updateState} />)`?
So that `if`s wouldn't be needed at all and params would be directly passed to the component (if they are defined or not)?